data/en/qcas_0018.json

```diff
@@ -41,8 +41,8 @@
             "id": "qcas-section",
             "title": "QCAS",
             "groups": [{
-                "id": "introduction-group",
-                "title": "Introduction",
+                "id": "qcas",
+                "title": "",
```

Comment:
I don't really like this, it should really not be defined. but given validator doesn't currently allow this 🤷‍♂️

Reply:
Agreed.